pkg/uevent/queue.go

```diff
@@ -68,42 +69,25 @@ func newEventQueue() *eventQueue {
 //   Update()
```

Comment:
we can remove these comments. They were used only while discussing on the video call